iot_control/models/iot_partner.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 thingsintouch.com
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# Copyright 2022 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PartnerIoT(models.Model):
+    _name = "iot.partner"
```

Comment:
I would not call it partner, as at the end it can be related to other kind of records.